applications/directory/src/components/ProfileReview.js

```diff
@@ -91,40 +92,74 @@ const ProfileReview = ({ onSubmit }) => {
           {submissionError}
         </Alert>
       )}
-      <Card elevation={2}>
+      <Card
+        elevation={2}
+        sx={{
+          width: ['auto', '80%'],
+          margin: '0 auto',
+          boxShadow: '-0.625rem 0.625rem 1.25rem 0 rgba(30, 30, 30, 0.05)' // the same shadow as in a few other places
+        }}
+      >
         <form noValidate onSubmit={handleSubmit}>
-          <Box sx={{ width: '75%' }}>
-            <Typography variant="h3" component="h3">
+          <Box sx={{ marginBottom: '0.875rem' }}>
+            <Typography
+              variant="h2"
+              sx={{
+                // these h2 styles are the same for 4 more headers (2 in this component and 2 in 2 other components)
+                fontSize: '1.25rem',
+                fontFamily: 'gira-sans, sans-serif',
+                fontWeight: 500,
+                marginBottom: '0.875rem',
+                textAlign: 'center'
+              }}
+            >
               Choose what do you want to do with the artist profile:
             </Typography>
-            <RadioGroup
-              row
-              aria-label="Approval status"
-              name="row-radio-buttons-group"
-              value={values.approvalStatus}
+
+            <Box
+              sx={{
+                marginBottom: '0.875rem' // this margin is all over the place on the review page (at least 5 different places)
```

Comment:
Maybe use a variable as we discussed (your idea) where `marginBottom: '0.875rem'` is used?

Reply:
done